examples/c/5_series/stream_imu/README.md

```diff
@@ -0,0 +1,188 @@
+# 5 Series IMU Streaming Example (C)
```

Comment:
At this point, what does stream_imu do that the device specific examples don't?

If there is something special about it, I would note it in the README to explain why a user might want to use this one instead of the device specific example. Otherwise, maybe it is safe to just remove this example?